README.md

```diff
@@ -64,6 +64,7 @@ There are three types of policies:
 We currently have the following policies in place:
 
 * rate-limiting when too many emails (`config.limits.maxEmails` defaults to 3) have been sent to the same email address in a given time period (`config.limits.rateLimitIntervalSeconds` defaults to 15 minutes)
+* rate-limiting when too many request for checking account status (`config.limits.maxAccountStatusCheck`) have been sent from the same ip address during period (`config.limits.rateLimitIntervalSeconds` defaults to 15 minutes)
```

Comment:
I feel like this could be slightly clearer about what we're guarding against, e.g. "too many requests to look up account status by email address".  We won't rate-limit attempts to look up account status by e.g. uid or sessionToken.